mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/mail/task/Task.java

```diff
@@ -55,6 +59,44 @@ static Result combine(Result result1, Result result2) {
         return Result.PARTIAL;
     }
 
+
+    class TaskId {
```

Comment:
If TaskId is just a UUID why introduce a class wrapping it and not using directly UUID ?

Reply:
Strong typing?